### ISSUE TITLE
Test Suite fails due to Railtie not being loaded

### ISSUE THREAD

**Issue by wfleming** [alexreisner/geocoder#780]
I just forked Geocoder, and after bundling `rake test` failed immediately with the error:

`'<class:Place>': undefined method 'geocoded_by' for Place:Class (NoMethodError)`

Not seeing any mention of this elsewhere in the issues or current PRs, I feel like I've probably missed something very obvious, but in case I haven't, I [changed the test_helper](https://github.com/wfleming/geocoder/compare/fix-test-suite) in my own fork to explicitly initialize the Railtie. My belief from my investigation was that this was happening because the Railtie wasn't actually being initialized in the test env because Rails & ActiveRecord were stubbed.

If this is actually an issue, I can open up a PR from [my branch](https://github.com/wfleming/geocoder/compare/fix-test-suite) if you like (or you can just go ahead & pull it in if you want since it's a one-liner). If I missed something, and this was unnecessary, then…sorry!

**Comment by alexreisner** [comment]
Sorry about that. Can you pull, rebase, and try again? I'm thinking 1fc56aca2a4806a2242f772ff350a8f5bc682c28 fixes this.

**Comment by wfleming** [comment]
That does indeed fix it, thanks.

Is doing it this way the best solution though? It does seem like it'll end up getting called twice in an actual Rails app with this (once in 1fc56aca2a4806a2242f772ff350a8f5bc682c28, and once by the [`initializer`](https://github.com/alexreisner/geocoder/blob/1fc56aca2a4806a2242f772ff350a8f5bc682c28/lib/geocoder/railtie.rb#L7-L10).

Doesn't seem like a huge deal since it won't result in any different results, but it does seem a bit odd. I *think* the only situation where you need the explicit call `insert` is when Rails is stubbed, so maybe calling it in the `test_helper.rb` would be a bit cleaner?

**Comment by alexreisner** [comment]
I totally agree, both in that it's not an ideal solution and that I'm not 100% sure when the explicit `insert` is needed. Do you think you can confirm whether putting it in `test_helper.rb` solves the problem, and if so, open a pull request?

**Comment by wfleming** [comment]
I can already confirm it works, since that's what I did while writing #779 (I just didn't commit it on that branch). Will open PR in a moment.
